app/retry.js

```diff
@@ -0,0 +1,16 @@
+const { totalRetries } = require('./config')
```

Comment:
There is no config value named `totalRetries` so default is `undefined` so will never attempt retry.